src/rustfmt_diff.rs

```diff
@@ -88,6 +101,7 @@ pub fn print_diff<F>(diff: Vec<Mismatch>, get_section_title: F)
     where F: Fn(u32) -> String
 {
     let mut t = term::stdout().unwrap();
+    let isatty = unsafe { libc::isatty(libc::STDOUT_FILENO) } != 0;
```

Comment:
Can we use something in libterm for this? `supports_attr` maybe? Seems kind of wrong to need a libc call here. If its not possible you should file an issue against libterm.